requirements.txt

```diff
@@ -74,7 +76,7 @@ pytest-sugar==0.9.4
 pytest-variables==1.9.0
 pytest-xdist==2.5.0
 python-dateutil==2.8.2
-python-dotenv==0.19.2
+python-dotenv==0.20.0
```

Comment:
🔥 I think we don't need this anymore

Reply:
I've removed it from here, but it still gets installed as a transitive dependency from fsd_utils as it's in the `setup.py` there

Reply:
I think it is still useful as per https://flask.palletsprojects.com/en/2.1.x/cli/#environment-variables-from-dotenv so we can inject from a .env file where that's useful.